packages/api/src/beacon/routes/beacon/pool.ts

```diff
@@ -94,6 +102,15 @@ export type Api = {
    */
   submitPoolVoluntaryExit(exit: phase0.SignedVoluntaryExit): Promise<void>;
 
+  /**
+   * Submit SignedBLSToExecutionChange object to node's pool
+   * Submits SignedBLSToExecutionChange object to node's pool and if passes validation node MUST broadcast it to network.
+   * @param requestBody
+   * @returns any BLSToExecutionChange is stored in node and broadcasted to network
+   * @throws ApiError
+   */
+  submitPoolBlsToExecutionChange(blsToExecutionChange: capella.SignedBLSToExecutionChange): Promise<void>;
```

Comment:
@g11tech  looks like we'll submit in batches for next devnets https://github.com/ethereum/beacon-APIs/pull/279

Reply:
thanks @tuyennhv ! :slightly_smiling_face: updated